script/build-libgit2-static.sh

```diff
@@ -16,4 +16,5 @@ cmake -DTHREADSAFE=ON \
       -DCMAKE_INSTALL_PREFIX=../install \
       .. &&
 
-cmake --build .
+cmake --build . &&
```

Comment:
You can use `cmake --build . --target install` as in https://github.com/libgit2/libgit2/blob/f4633791104fa5be75115fd4c7fab0c097da345e/script/cibuild.sh#L27